patches/minecraft/net/minecraft/client/renderer/RenderGlobal.java.patch

```diff
@@ -93,29 +93,119 @@
                  {
                      this.field_175009_l.add(renderchunk4);
                  }
-@@ -1253,6 +1268,13 @@
+@@ -1253,13 +1268,15 @@
  
      public void func_174976_a(float p_174976_1_, int p_174976_2_)
      {
-+        net.minecraftforge.client.IRenderHandler renderer = this.field_72769_h.field_73011_w.getSkyRenderer();
-+        if (renderer != null)
-+        {
-+            renderer.render(p_174976_1_, field_72769_h, field_72777_q);
-+            return;
-+        }
-+
++        int renderPass = net.minecraftforge.fml.client.FMLClientHandler.instance().renderSky(p_174976_1_);
```

Comment:
Can you use `SkyRenderPass` instead of `int` here? I think it would make things clearer.

Reply:
No I can't, because there are several vanilla renderers for 'SunMoonStars' pass.